src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -20,4 +23,32 @@ public LineResponse saveLine(LineRequest request) {
         Line persistLine = lineRepository.save(request.toLine());
         return LineResponse.of(persistLine);
     }
+
+    @Transactional(readOnly = true)
+    public List<LineResponse> findAllLines() {
+        List<Line> Lines = lineRepository.findAll();
+
+        return Lines.stream()
+                .map(LineResponse::of)
+                .collect(Collectors.toList());
+    }
+
+    @Transactional(readOnly = true)
+    public LineResponse findLine(final Long id) {
+         Line line = lineRepository.findById(id)
+                 .orElseThrow(() -> new LineNotFoundException());
+
+        return LineResponse.of(line);
+    }
+
+    public void updateLine(final Long id, final LineRequest request) {
+        Line line = lineRepository.findById(id)
+                .orElseThrow(() -> new LineNotFoundException());
+
+        line.update(new Line(request.getName(), request.getColor()));
```

Comment:
```suggestion
        line.update(request.toLine());
```

만들어져있는 팩토리 메서드를 활용할 수 있지 않을까요?

Reply:
앗 수정하겠습니다.